homeassistant/components/dio_chacon/entity.py

```diff
@@ -0,0 +1,32 @@
+"""Base entity for the Dio Chacon entity."""
+
+from dio_chacon_wifi_api import DIOChaconAPIClient
+
+from homeassistant.helpers.device_registry import DeviceInfo
+
+from .const import DOMAIN, MANUFACTURER
+
+
+class DioChaconEntity:
```

Comment:
Since everything is push based, should we add `_attr_should_poll = False`?

Reply:
yes